system/src/system_sleep.cpp

```diff
@@ -63,8 +68,12 @@ static bool system_sleep_network_suspend(network_interface_index index) {
 }
 
 static int system_sleep_network_resume(network_interface_index index) {
+#if PLATFORM_GEN == 2
+    SPARK_WLAN_SLEEP = 0;
```

Comment:
Let's add a comment explaining why this was needed.

Reply:
Added.